com.unity.render-pipelines.core/Runtime/Debugging/VolumeDebugSettings.cs

```diff
@@ -4,26 +4,29 @@
 using System.Reflection;
 using UnityEditor;
 
-namespace UnityEngine.Rendering.HighDefinition
+namespace UnityEngine.Rendering
 {
     /// <summary>
     /// Volume debug settings.
     /// </summary>
-    public class VolumeDebugSettings
+    public abstract class VolumeDebugSettings<T> : IVolumeDebugSettings
+        where T : MonoBehaviour, IAdditionalData
     {
         /// <summary>Current volume component to debug.</summary>
-        internal int selectedComponent = 0;
+        public int selectedComponent { get; set; } = 0;
+
         int m_SelectedCamera = 0;
 
-        internal int selectedCameraIndex
+        /// <summary>Selected camera index</summary>
+        public int selectedCameraIndex
         {
             get
             {
 #if UNITY_EDITOR
-                if (m_SelectedCamera < 0 || m_SelectedCamera > cameras.Count + 1)
+                if (m_SelectedCamera < 0 || m_SelectedCamera > additionalCameraDatas.Count + 1)
```

Comment:
I know thiw iw not your change but, `m_SelectedCamera` make me think about a camera object not an index. Should we rename it `m_SelectedCameraIndex`?